Dockerfile

```diff
@@ -1,49 +1,43 @@
 #default image to ubuntu + install rocm
-ARG BASEIMAGE=ubuntu:20.04
-ARG ROCM_PRE=0
-#ARG IMG_VER=$([[ $BASEIMAGE == "ubuntu:20.04" ]]; echo $?)
+ARG BASEIMAGE=rocm/miopen:ci_5450cc
 
-FROM ubuntu:20.04 as dtuna-ver-0
+#FROM ubuntu:20.04 as dtuna-ver-0
+FROM $BASEIMAGE as dtuna-ver-0
 #install rocm
 ARG ROCMVERSION=
-ARG OSDB_BKC_VERSION='12969'
+ARG OSDB_BKC_VERSION=
+#'12969'
```

Comment:
Do we need comment? line

Reply:
It's just a note. BKC can take some time to look up. Though it has since been updated.